reana_client/cli/workflow.py

```diff
@@ -455,9 +473,7 @@ def add_verbose_data_from_response(response, verbose_headers,
                 err=True)
 
 
-@workflow_execution_group.command(
-    'logs',
-    help='Get workflow logs.')
```

Comment:
Add some body, e.g.

Retrieve logs of running workflow.  Note that only finished steps of the workflow are returned, the logs of the currently processed step is not returned until it is finished.

Reply:
... or better said in the usual optics:

The `logs` command permits to retrieve....